pkg/sql/jobs_profiler_bundle.go

```diff
@@ -86,7 +87,7 @@ func (e *ExecutionDetailsBuilder) WriteExecutionDetail(
 				chunk = chunk[:chunkSize]
 			} else {
 				// This is the last chunk we will write, assign it a sentinel file name.
-				chunkFileName = chunkFileName + "_final"
+				chunkFileName = chunkFileName + finalChunkSuffix
```

Comment:
maybe this is just my pre-coffee brain, but does this end up being encoded as filename_final#num instead of filename#final?